### PR TITLE
(MODULES-6809) Clarify debugging with Puppetserver log

### PR DESCRIPTION
**Pull request by eimlav**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -139,19 +139,36 @@ This example tells the master the location of the Satellite server (`https://pup
...
 
 An example of a sucessful SSL authentication (note the '201'):
 
-~~~puppet
+```puppet
 10.32.125.164 - - [03/Oct/2015:16:06:00 -0700] "POST /api/reports HTTP/1.1" 201 554 "-" "Ruby"
-~~~
+```
+
+The Puppetserver log file is located at `/var/log/puppetlabs/puppetserver/puppetserver.log` on your Puppet server.
```
Comment (clairecadman) [inline README.md]: Either Puppet server or puppetserver

File: README.md [hunk 2]
```diff
@@ -139,19 +139,36 @@ This example tells the master the location of the Satellite server (`https://pup
 
 ## Debugging
 
-In addition to looking through the usual reports in the Puppet Enterprise Console, you can also view the Satellite API log file, which may provide clues as to what a particular issue may be. This file is located at `/var/log/httpd/foreman-ssl_access_ssl.log` on your Satellite server.
+In addition to looking through the usual reports in the Puppet Enterprise Console, also review the Satellite API log and the Puppetserver log.
```
Comment (clairecadman) [inline README.md]: Are the Satellite API log and Puppet server log also the PE console? If so, maybe just "You can view the Satellite API log and the Puppet server log in the Puppet Enterprise console".

(lowercase for 'console').

File: README.md [hunk 3]
```diff
@@ -139,19 +139,36 @@ This example tells the master the location of the Satellite server (`https://pup
...
+
+The Puppetserver log file is located at `/var/log/puppetlabs/puppetserver/puppetserver.log` on your Puppet server. 
+
+An example of a DH PARAMETER failure:
+
+```puppet
+2018-03-04 15:16:17,161 ERROR [qtp1111094392-103] [puppetserver] Puppet Could not send report to Satellite: Could not generate DH keypair
+```
+
+The above error can be resolved by adding a DH PARAMETER block to the custom certificate on the Satellite server.
```
Comment (clairecadman) [inline README.md]: Reword to active voice "You can resolve this error by adding a DH PARAMETER block to the custom certificate on the Satellite server."

File: README.md (hunk 2 above)
Comment (eimlav) [reply README.md]: The Satellite API log and Puppet server log are separate files that must be manually accessed. See lines 144 and 158. Will I just leave this as is then?

**Comment by eimlav** [comment]
@clairecadman Changes have been made. Can you re-review? Thanks!

File: README.md (hunk 2 above)
Comment (clairecadman) [reply README.md]: I'd change this explain you want to look at these files, kind of like the original paragraph. For example: "In addition to the reports in the Puppet Enterprise Console, the Satellite API log and the Puppet server log can help you debug issues".

**Comment by clairecadman** [comment]
@eimlav I added one more comment, but otherwise looks good.

**Pull request merged**